libcudacxx/include/cuda/std/__memory/compressed_pair.h

```diff
@@ -0,0 +1,235 @@
+// -*- C++ -*-
```

Comment:
There's also compressed_pair.h in __mdspan, please unify if possible.

Reply:
There is and I want to burn that part with fire. But today is not that day